modules/kind/kind-image-preload.mk

```diff
@@ -26,22 +26,21 @@ endif
 
 ##########################################
 
-images := $(images_$(HOST_ARCH))
-images_files := $(foreach image,$(images),$(subst :,+,$(image)))
+IMAGE_ARCH ?= $(HOST_ARCH)
```

Comment:
doesn't seem useful? I don't see a use-case where having `HOST_ARCH` and `IMAGE_ARCH` different would help (or even work)